File: Python Word Examples/TablesApiDemo/Rows/addTableRow.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.TablesApi(configuration)

file_name = 'Template.docx'
table_path = 'Section/0/Body/0/Table/6'
extra = {'index_of_table_row': 0,'cells_count': 5,'folder': "input", 'dest_file_path': 'output/AddTableRow.docx'}
result = api.add_table_row(file_name, table_path, **extra)

File: PHP Word Examples/TablesApiDemo/Rows/addTableRow.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\TablesApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new TablesApi($configuration);

$fileName="AddTableRow_1_Temp.docx";
$table_path = 'Section/0/Body/0/Table/0';
$index_of_table_row = 0;
$cells_count = 4;
$dest_file_path = "Tables/AddTableRow_1.docx";
$folder="Tables";
$password =null;
$storage = null;
$apiInstance->addTableRow($fileName, $table_path, $dest_file_path,$folder , $storage, $index_of_table_row, $cells_count, $password);